ehsm_kms_service/ehsm_kms_params.js

```diff
@@ -106,7 +106,29 @@ const ehsm_kms_params = {
       maxLength: _6kb_length,
       minLength: 1
     },
-  }
+  },
+  AsymmetricEncrypt: {
+    cmk_base64: {
+      type: 'string',
+      minLength: 1
+    },
+    plaintext: {
+      type: 'string',
+      maxLength: _6kb_length,
+      minLength: 1
+    },
+  },
+  AsymmetricDecrypt: {
+    cmk_base64: {
+      type: 'string',
+      minLength: 1
+    },
+    ciphertext_base64: {
+      type: 'string',
+      maxLength: _6kb_length,
```

Comment:
The Length is not 6KB,  I need to check.

Reply:
It will change to 384.

Reply:
for the maxlength in the nodejs,  previous patches also have the simlar issue, i remember that you told me that you want to refine the together?